numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4106,16 +4106,16 @@ array_may_share_memory(PyObject *NPY_UNUSED(ignored), PyObject *args, PyObject *
 static PyObject *
 normalize_axis_index(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwds)
 {
-    static char *kwlist[] = {"axis", "ndim", NULL};
+    static char *kwlist[] = {"axis", "ndim", "msg_prefix", NULL};
     int axis;
     int ndim;
+    PyObject *msg_prefix = Py_None;
```

Comment:
Here, again, I think you cannot just use `Py_None`. Also solve by initlializing to `NULL`?